sdk/identity/Azure.Identity/tests/ManagedIdentityCredentialTests.cs

```diff
@@ -376,7 +376,7 @@ public async Task VerifyInitialImdsConnectionTimeoutHonored()
         {
             using var server = new TestServer(async context =>
             {
-                await Task.Delay(8000);
```

Comment:
Was 8000 tied to some default timeout in MSI endpoint discovery?

Reply:
@schaabs Do you recall why 8 seconds was chosen?